src/sentry/feedback/endpoints/project_feedback_details.py

```diff
@@ -6,20 +6,29 @@
 from sentry.api.api_owners import ApiOwner
 from sentry.api.api_publish_status import ApiPublishStatus
 from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.project import ProjectPermission
 from sentry.api.serializers.base import serialize
 from sentry.feedback.endpoints.base import ProjectFeedbackEndpoint
 from sentry.feedback.models import Feedback
 from sentry.feedback.serializers import FeedbackSerializer
 from sentry.models.project import Project
 
 
+class ProjectFeedbackDetailsPermission(ProjectPermission):
+    scope_map = {
+        "GET": ["project:read", "project:write", "project:admin"],
+        "DELETE": ["project:read", "project:write", "project:admin"],
```

Comment:
do we need to remove project_admin from this?

Reply:
we're adding `project:read` and `project:write` to the list. `project_admin` is in it by default